prow/cluster/plank_deployment.yaml

```diff
@@ -29,13 +29,22 @@ spec:
         role: prow
       containers:
       - name: plank
-        image: gcr.io/k8s-prow/plank:0.17
+        image: gcr.io/k8s-prow/plank:0.18
+        args:
+        - "--dry-run=false"
+        - "--github-bot-name=k8s-ci-robot"
```

Comment:
This seems like it can move in the prow config since it's used by hook too (and I guess whatever other component in the future needs to use the bot).